docs/operations/export_api.md

```diff
@@ -217,6 +217,23 @@ The export server certificate is valid for 7 days after which it is rotated by d
 #### External link certificates
 The external link certificates are associated with the Ingress/Route that points to the service created by the KubeVirt operator. The CA that signed the Ingress/Route will part of the certificates. 
 
+### TTL (Time to live) for an Export
+For various reasons (security being one), users should be able to specify a TTL for the VMExport objects that limits the lifetime of an export.  
+This is done via the `ttlDuration` field which accepts a k8s duration.
```

Comment:
It may be useful to link to a reference for how to format durations in kubernetes

Reply:
done

Reply:
gentle ping @cwilkers